source/intel.js

```diff
@@ -1711,6 +1719,238 @@ NeptunesPride.npui.StarInspector = function () {
 };
 
 
+//research screen replacement
+//
+const overhaul_research_panel = () => {
```

Comment:
I am going to branch this and test this but in the future it would be helpful if you supplied screenshots of what changes this occurs...

Also, I generally try to avoid re-writing whole panels like this. But, if what you did is very helpful it is okay. You can generally achieve the same thing by adding new panels rather then editing old ones. 

Changing these panels makes it so if Jay does decide to change the panel in any way this will break it. 

Or possibly use query selectors to change existing content. I will review you stuff though and probably push it along for now.